src/cargo/core/features.rs

```diff
@@ -786,6 +786,7 @@ unstable_cli_options!(
     target_applies_to_host: bool = ("Enable the `target-applies-to-host` key in the .cargo/config.toml file"),
     trim_paths: bool = ("Enable the `trim-paths` option in profiles"),
     unstable_options: bool = ("Allow the usage of unstable options"),
+    update_precise_breaking: bool = ("Allow `update --precise` to do breaking upgrades"),
```

Comment:
Why a new unstable option rather than `unstable_options`?

Reply:
Because since there isn't any difference between the breaking command `cargo update foo --precise 2.0.1` and its non-breaking counterpart, I thought we needed a feature.

Should the breaking instead be `cargo update foo --breaking --precise 2.0.1`?

Or is it fine to just use `unstable-options` to allow the breaking update?

Reply:
I view this as a new supported value within the option which could be used with `-Zunstable-options`.  If there is doing more than turning error cases into success cases, then we might want to consider a more explicit opt-in.

Reply:
Fixed.